cypress/e2e/article.cy.js

```diff
@@ -1,24 +1,77 @@
 /// <reference types='cypress' />
 /// <reference types='../support' />
+import ArticlePage from '../support/pages/article.pageObject';
+import SignInPageObject from '../support/pages/signIn.pageObject';
+import HomePageObject from '../support/pages/home.pageObject';
+import faker from 'faker';
+import '../support/commands';
 
-describe('Article', () => {
-  before(() => {
+const signInPage = new SignInPageObject();
+const homePage = new HomePageObject();
+const articlePage = new ArticlePage();
+
+describe('Article Page', () => {
+  let user;
+  let title;
+  let description;
+  let content;
+  let tags;
 
+  before(() => {
+    cy.task('db:clear');
   });
 
   beforeEach(() => {
-    cy.task('db:clear');
+    cy.task('generateUser').then((generateUser) => {
+      user = generateUser;
+
+      cy.registerUser(user.email, user.username, user.password);
+      cy.visit('/#/login');
+      signInPage.login(user.email, user.password);
+      homePage.clickNewArticle();
+
+      title = faker.lorem.words(2);
+      description = faker.lorem.words(3);
+      content = faker.lorem.words(4);
+      tags = ['tag1', 'tag2', 'tag3'];
+      const tagsString = tags.join(', ');
+
+      articlePage.fillTitle(title);
+      articlePage.fillDescription(description);
+      articlePage.fillContent(content);
+      articlePage.fillTags(tagsString);
+      articlePage.submitArticle();
```

Comment:
After submitting the article form, there is no assertion to verify if the article has been created successfully. Consider adding an assertion after form submission to check if the article was created successfully.

Reply:
All assertions are in tests
![e2e_18](https://github.com/mate-academy/qa_cypress_e2e_project/assets/13405550/e2019dde-eebd-490c-8e13-f9410b821f4d)